extensions/ql-vscode/src/remote-queries/view/ExportButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.a`
```

Comment:
If you make the following changes I think you get the same result but it hopefully makes a bit more sense:
- Use a `<button>` instead of `<a>` (`styled.button...`)
- Set `border: 0;`
- Remove `className="monaco-button monaco-text-button"` from L17

Reply:
Thank you! That worked 🎉